wolfssl/ssl.h

```diff
@@ -3805,6 +3830,38 @@ WOLFSSL_API int wolfSSL_sk_CONF_VALUE_push(WOLF_STACK_OF(WOLFSSL_CONF_VALUE)* sk
         WOLFSSL_CONF_VALUE* val);
 #endif /* OPENSSL_ALL || HAVE_STUNNEL || WOLFSSL_NGINX || WOLFSSL_HAPROXY || OPENSSL_EXTRA || HAVE_LIGHTY */
 
+#if defined(OPENSSL_ALL) && !defined(NO_FILESYSTEM) && !defined(NO_WOLFSSL_DIR)
```

Comment:
General question - do we need to export all these functions as wolfSSL API, or are they just called internally by other wolfSSL / compatibility functions?

Reply:
I thought I had to follow similar functions, such as wolfSSL_sk_X509_NAME_xx and so on. That is why those functions are exported as wolfSSL API now. As you are pointing it out, that is true that those functions are only called internally by other wolfSSL / compatibility functions. 
Therefore, I would move them to internal.c.  In that case, Am I able to still use "wolfSSL_sk_XXX" name convention, or should we use different name convention? It seems that there are few those function name.

Reply:
Moved those functions to internal.c, removed those API testing in api.c.

Reply:
Thanks, keeping those functions internal unless we know an external user will need to use them, will make maintenance easier.

Reply:
Moved struct body definition to internal.h too